libs/rpcd-mod-luci/src/luci.c

```diff
@@ -696,7 +696,11 @@ rpc_luci_parse_network_device_sys(const char *name, struct ifaddrs *ifaddr)
 		for (n = 0, p += strlen("DEVTYPE=");; n++) {
 			if (p[n] == '\0' || p[n] == '\n') {
 				p[n] = 0;
-				blobmsg_add_string(&blob, "devtype", p);
+				if (!strcmp(p, "bond")) {
```

Comment:
netifd opted for device type "bonding" instead of Linux device type "bond". To be as consistent as possible throught luci I remove the os type as soon as possible.

Reply:
I'd rather stick with the kernel name and translate in LuCI as needed.